src/components/summary-list/index.md

```diff
@@ -46,6 +46,12 @@ You can add ‘row actions’ to a summary list. For example, you can help users
 
 When navigating visually, the borders above and below each row help to show which row action is tied to which piece of information.
 
+<strong class="govuk-tag govuk-tag--grey">WCAG 2.2</strong> Keep card and row actions as ‘inline links’. This is to make sure that sets of actions reflow properly on different screen sizes. This relates to WCAG 2.2 success criterion [2.5.8 Target Size (minimum)](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html).
+
+If a user decides to go back to a previous answer through a card or row action, make sure information they have already entered is pre-populated.
+
+Do not pre-populate if the information is no longer valid, or when pre-populating would be a major safety or security concern. This is to comply with WCAG 2.2 success criterion [3.3.7 Redundant Entry](https://www.w3.org/WAI/WCAG22/Understanding/redundant-entry.html).
```

Comment:
...And here's an example of 2 messages put under "one tag"

Reply:
Decided that omitted the 2nd WCAG tag doesn't read right, so going to try repeating and see how it works with others.

```suggestion
<strong class="govuk-tag govuk-tag--grey">WCAG 2.2</strong> Keep card and row actions as ‘inline links’. This is to make sure that sets of actions reflow properly on different screen sizes. This relates to WCAG 2.2 success criterion [2.5.8 Target Size (minimum)](https://www.w3.org/WAI/WCAG22/Understanding/target-size-minimum.html).

If a user decides to go back to a previous answer through a card or row action, make sure information they have already entered is pre-populated.

<strong class="govuk-tag govuk-tag--grey">WCAG 2.2</strong> Do not pre-populate if the information is no longer valid, or when pre-populating would be a major safety or security concern. This is to comply with WCAG 2.2 success criterion [3.3.7 Redundant Entry](https://www.w3.org/WAI/WCAG22/Understanding/redundant-entry.html).
```